app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, old_file, path = command.split()
+    path_dirs = path.split("/")
```

Comment:
Do not use any specific directory delimiter in your code, your code should be platform-agnostic.